db/version_set.cc

```diff
@@ -437,11 +435,12 @@ namespace {
 class LevelFileNumIterator : public InternalIterator {
  public:
   LevelFileNumIterator(const InternalKeyComparator& icmp,
-                       const LevelFilesBrief* flevel)
+                       const LevelFilesBrief* flevel, bool should_sample)
       : icmp_(icmp),
         flevel_(flevel),
         index_(static_cast<uint32_t>(flevel->num_files)),
-        current_value_(0, 0, 0) {  // Marks as invalid
+        current_value_(0, 0, 0),
+        should_sample_(should_sample) {  // Marks as invalid
```

Comment:
The comment "Marks as invalid" should be against current_value_  (i.e. the FileDescriptor instance).